libraries/utilities/include/mcrl2/utilities/detail/utility.h

```diff
@@ -0,0 +1,65 @@
+// Author(s): Maurice Laveaux
+// Copyright: see the accompanying file COPYING or copy at
+// https://github.com/mCRL2org/mCRL2/blob/master/COPYING
+//
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef CLHTPP_DETAIL_UTILITY_H_
+#define CLHTPP_DETAIL_UTILITY_H_
```

Comment:
I would suggest to make the filename more descriptive and also bring these include guards in line with the standard format.

Reply:
Moved the header from detail/utility.h to power_of_two.h and updated the include guards.